### PR TITLE
add pom spring-boot-starter-test and move test unit

### PR DESCRIPTION
**Pull request by Gdk666**
Fixes #ISSUSE_ID

Changes proposed in this pull request:
- add pom spring-boot-starter-test and move test unit

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/docs/community/contributor)

### CODE REVIEW

**Comment by Gdk666** [comment]
add pom spring-boot-starter-test and move test unit

File: hippo4j-config/src/test/java/cn/hippo4j/config/toolkit/BeanUtilTest.java [hunk 1]
```diff
@@ -0,0 +1 @@
...
 * limitations under the License.
 */

package cn.hippo4j.config.toolkit;

/**
 * BeanUtil Test
 */
public class BeanUtilTest {
}
```
Comment (agentart) [inline hippo4j-config/src/test/java/cn/hippo4j/config/toolkit/BeanUtilTest.java:1]: 这些变更是否有问题

Comment (Gdk666) [reply hippo4j-config/src/test/java/cn/hippo4j/config/toolkit/BeanUtilTest.java:1]: 没问题，修改包路径与其他模块统一

Comment (Gdk666) [reply hippo4j-config/src/test/java/cn/hippo4j/config/toolkit/BeanUtilTest.java:1]: config模块，引入spring-boot-starter-test

**Pull request merged**